langchain/cli/create_repo/templates/poetry/pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "____project_name"
+version = "0.0.1"
+description = ""
+authors = ["____author_name <____author_email>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "____project_name_identifier"}]
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+langchain = "^____langchain_version"
```

Comment:
I don't think this is a good idea, langchain should be a peer dependency, otherwise people will end up with incompatible version ranges for langchain in their projects as soon as they use more than 1 of these packages?